languages/tree-sitter-stack-graphs-typescript/rust/lib.rs

```diff
@@ -17,9 +17,10 @@ pub mod tsconfig;
 pub mod util;
 
 /// The stacks graphs tsg path for this language.
-pub const STACK_GRAPHS_TSG_PATH: &str = "src/stack-graphs.tsg";
+pub const STACK_GRAPHS_TSG_PATH: &str = "./stack-graphs.tsg";
```

Comment:
```suggestion
pub const STACK_GRAPHS_TSG_PATH: &str = "src/stack-graphs.tsg";
```

This should be relative to the crate root for VSCode to handle it correctly when running `cargo test` e.g.

Reply:
good catch!